### DISCUSSION TITLE
The right version of PyQt5 doesn't seem to be on PyPI anymore

### DISCUSSION THREAD

**Discussion by ajsirota** [mu-editor/mu#1501] [Q&A]
When I try to run a `pip install -e ".[dev]"`, `pip` complains that it can't find `PyQt5==5.13.2`.
```
(.venv) ajs256@greeble mu %  pip install -e ".[dev]" 
Obtaining file:///Users/ajs256/code/mu
Collecting jupyter-client<6.2,>=4.1
  Using cached jupyter_client-6.1.12-py3-none-any.whl (112 kB)
Collecting qtconsole==4.7.7
  Using cached qtconsole-4.7.7-py2.py3-none-any.whl (118 kB)
Collecting adafruit-board-toolkit>=1.0.1
  Using cached adafruit-board-toolkit-1.0.1.tar.gz (33 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
    Preparing wheel metadata ... done
Collecting pyserial>=3.5
  Using cached pyserial-3.5-py2.py3-none-any.whl (90 kB)
Collecting nudatus>=0.0.3
  Using cached nudatus-0.0.5-py3-none-any.whl (5.3 kB)
Collecting flake8>=3.8.3
  Using cached flake8-3.9.1-py2.py3-none-any.whl (73 kB)
Collecting appdirs>=1.4.3
  Using cached appdirs-1.4.4-py2.py3-none-any.whl (9.6 kB)
Collecting semver>=2.8.0
  Using cached semver-2.13.0-py2.py3-none-any.whl (12 kB)
Collecting virtualenv>=16.0.0
  Using cached virtualenv-20.4.4-py2.py3-none-any.whl (7.2 MB)
Collecting wheel
  Using cached wheel-0.36.2-py2.py3-none-any.whl (35 kB)
ERROR: Could not find a version that satisfies the requirement PyQt5==5.13.2 (from mu-editor==1.1.0b4) (from versions: 5.14.0, 5.14.1, 5.14.2, 5.15.0, 5.15.1, 5.15.2, 5.15.3, 5.15.4)
ERROR: No matching distribution found for PyQt5==5.13.2 (from mu-editor==1.1.0b4)
```

What can I do to fix this?

**Comment by carlosperate** [comment]
What version of your operating system, Python, and pip are you using?

**Reply by ajsirota** [reply]
* macOS 11.1 on Intel
* Python 3.8.9
* pip 20.2.3

**Reply by carlosperate** [reply]
Thanks @ajs256.

I just tested in Ubuntu with Python 3.8.9 and pip 20.2.3 and in macOS with Python 3.8.2 and pip 20.2.3, and in both cases it worked fine. If you try again today, do you still have the issue?

**Reply by ajsirota** [reply]
@carlosperate, yep. I still have the issue.

**Reply by carlosperate** [reply]
Weird. Thanks for checking again. 
Could you pip install with the verbose flag set to the maximum level and copy paste the result here?

**Reply by ajsirota** [reply]
[Here's the output as a gist](https://gist.github.com/ajs256/ffc0f5da57484cc7eec600feba49c23b)

**Reply by ajsirota** [reply]
Should I go to the pip issues/forum for this?

**Reply by carlosperate** [reply]
Often it's an issue with the package itself, so let's see if we can figure that out first before opening a pip issue.

Thanks for the gist, although I don't any errors there, was that the full output?
If you are running it again maybe try `pip install -vvv PyQt5==5.13.2` first to see if it's enough to trigger it, and if it doesn't then `pip install -vvv -e .` would still include the PyQt dependencies without all the other developer deps.

**Reply by ajsirota** [reply]
There is an error in the output, but the Gist website truncates it before the necessary part is visible. I added a new file to the Gist, `truncated.txt`, that just has the error.

**Edit:** [Here it is as its own gist](https://gist.github.com/ajs256/27332e535444ecdfece4123327168e5c)
